locale/arc/LC_MESSAGES/django.po

```diff
@@ -942,3 +937,208 @@ msgstr "لم يتم العثور على نتائج البحث ل "
+msgstr "ما هو أعلى مستوى من التعليم؟"
+
+#: iogt/temp_ve/profiles/templates/profiles/done.html:63
+msgid "Home Page"
+msgstr "الصفحة الرئيسية"
+
+#: iogt/temp_ve/profiles/templates/profiles/editprofile.html:11
+msgid "Update your display name:"
+msgstr "تحديث اسم العرض الخاص بك:"
+
+#: iogt/temp_ve/profiles/templates/profiles/editprofile.html:28
+msgid "Update your gender:"
+msgstr "تحديث جنسك:"
+
+#: iogt/temp_ve/profiles/templates/profiles/editprofile.html:37
+msgid "Update where you live:"
+msgstr "تحديث المكان الذي تعيش فيه:"
+
+#: iogt/temp_ve/profiles/templates/profiles/editprofile.html:46
+msgid "Update your Education Level:"
+msgstr "تحديث مستوى التعليم الخاص بك:"
+
+#: iogt/temp_ve/profiles/templates/profiles/register.html:21
+msgid "CHOOSE A USERNAME"
+msgstr "اختر اسم المستخدم"
+
+#: iogt/temp_ve/profiles/templates/profiles/register.html:24
+#: iogt/templates/profiles/register.html:24
+msgid ""
+"This name you will use to log in and won't appear to other users. Only you "
+"will see this."
+msgstr "هذا الاسم الذي سوف يستخدم لتسجيل الدخول ولن يظهر للمستخدمين الآخرين. أنت فقط سوف نرى هذا."
+
+#: iogt/temp_ve/profiles/templates/profiles/register.html:28
+msgid "CHOOSE A 4-DIGIT PIN"
+msgstr "اختر PIN المكون من 4 أرقام"
+
+#: iogt/temp_ve/profiles/templates/profiles/register.html:36
+msgid "CHOOSE A DISPLAY NAME"
+msgstr "اختيار اسم العرض"
+
+#: iogt/temp_ve/profiles/templates/profiles/register.html:81
+msgid "ENTER YOUR MOBILE NUMBER"
+msgstr "أدخل رقم هاتفك المحمول"
+
+#: iogt/temp_ve/profiles/templates/profiles/register.html:90
+msgid "ENTER YOUR EMAIL ADDRESS"
+msgstr "أدخل عنوان بريدك الالكتروني"
+
+#: iogt/temp_ve/profiles/templates/profiles/viewprofile.html:33
+msgid "Gender:"
+msgstr "الجنس :"
+
+#: iogt/temp_ve/profiles/templates/profiles/viewprofile.html:40
+msgid "Location:"
+msgstr "الموقع:"
+
+#: iogt/temp_ve/profiles/templates/profiles/viewprofile.html:47
+msgid "Education Level:"
+msgstr "مستوى التعليم"
+
+#: iogt/templates/profiles/register.html:36
+msgid "Choose a display name"
+msgstr "اختيار اسم العرض"
+
+#: iogt/templates/profiles/register.html:45
+msgid "Select date of birth"
+msgstr "حدد تاريخ الميلاد"
+
+#: iogt/templates/profiles/register.html:54
+msgid "I identify my gender as:"
+msgstr "أحدد بلدي الجنسين على النحو التالي:"
+
+#: iogt/templates/profiles/register.html:63
+msgid "Where do you live?"
+msgstr "أين تسكن؟"
+
+#: iogt/templates/profiles/register.html:72
+msgid "What is your highest level of education?"
+msgstr "ما هو أعلى مستوى من التعليم؟"
+
+#: iogt/settings/base.py:429
+msgid "This field is required"
+msgstr "هذه الخانة مطلوبه"
+
+#: iogt/settings/base.py:430
+msgid "ARE YOU A"
+msgstr "هل أنت"
+
+#: iogt/settings/base.py:431
+msgid "Ensure this value has at least 4 characters (it has 1)"
+msgstr "ضمان هذه القيمة 4 أحرف على الأقل (له 1)"
+
+#: iogt/settings/base.py:432
+msgid "Type your comment here"
+msgstr "اكتب تعليقك هنا"
+
+#: iogt/settings/base.py:434
+msgid "1"
+msgstr "1 "
```

Comment:
please remove these numbers, they are pointless, and also please remove them from settings